subsys/bluetooth/controller/Kconfig

```diff
@@ -484,6 +487,14 @@ config BT_CTLR_CONN_RSSI
 	help
 	  Enable connection RSSI measurement.
 
+config BT_CTLR_LE_POWER_CONTROL
+	bool "LE Power Control Request Feature"
+	depends on BT_CTLR_LE_POWER_CONTROL_SUPPORT
+	default y if BT_TRANSMIT_POWER_CONTROL
```

Comment:
There is both a `default` here and a `select` in `BT_TRANSMIT_POWER_CONTROL`. Is it correct to have both?

Reply:
You are right, best case overkill, worst case source of kconfig issues in future. I will remove the "default y if".